src/main/java/org/mitre/dsmiley/httpproxy/ProxyServlet.java

```diff
@@ -593,6 +592,14 @@ protected String rewriteQueryStringFromRequest(HttpServletRequest servletRequest
     return queryString;
   }
 
+  /**
+   * Allow overrides of {@link javax.servlet.http.HttpServletRequest#getPathInfo()}.
+   * Useful when url-pattern of servlet-mapping (web.xml) requires manipulation.
+   */
+  protected String rewritePathInfoFromRequest(HttpServletRequest servletRequest, String pathInfo) {
```

Comment:
I think this method should not take pathInfo as a parameter but it should return servletRequest.getPathInfo()

Reply:
👍